WordPress/src/main/java/org/wordpress/android/models/JetpackSettingsModel.java

```diff
@@ -1,17 +1,27 @@
 package org.wordpress.android.models;
 
+import java.util.ArrayList;
+
 public class JetpackSettingsModel {
     public long localTableId;
     public boolean monitorActive;
     public boolean emailNotifications;
     public boolean wpNotifications;
+    public boolean jetpackProtectEnabled;
+    public final ArrayList<String> jetpackProtectWhitelist = new ArrayList<>();
 
     @Override
     public boolean equals(Object other) {
         if (!(other instanceof JetpackSettingsModel)) return false;
         JetpackSettingsModel otherModel = (JetpackSettingsModel) other;
         return monitorActive == otherModel.monitorActive &&
                 emailNotifications == otherModel.emailNotifications &&
-                wpNotifications == otherModel.wpNotifications;
+                wpNotifications == otherModel.wpNotifications &&
+                jetpackProtectEnabled == otherModel.jetpackProtectEnabled &&
+                whitelistMatches(otherModel.jetpackProtectWhitelist);
+    }
+
+    public boolean whitelistMatches(ArrayList<String> otherWhitelist) {
+        return otherWhitelist != null && jetpackProtectWhitelist.equals(otherWhitelist);
```

Comment:
I think the order will matter if we use `equals` for comparing the lists. It might be better to convert them to sets and then check the equality.

On a separate note, in it's current form, I don't think the `null` check is necessary, since the equals will just fail if `otherWhitelist` is null.